gruntfile.js

```diff
@@ -2,6 +2,7 @@ module.exports = function( grunt ) {
 
 	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
 	require( './.deployment' );
+	var dest = 'dist';
```

Comment:
Another idea is to put the var declaration above as a 'default' and if your `.deployment` file has a `dest` variable export in it, then it will be overwritten.